study-builder/studies/rarex/health-and-development.conf

```diff
@@ -2499,39 +2499,6 @@
           "blockType": "QUESTION",
           "shownExpr": """user.studies["rarex"].forms["ADD_PARTICIPANT"].questions["PARTICIPANT_RELATIONSHIP"].answers.hasOption("SELF")"""
         },
-        {
-          "titleTemplate": null,
-          "bodyTemplate": {
-            "templateType": "HTML",
-            "templateText": """<h3 class="question-prompt">$question</h4>""",
-            "variables": [
-              {
-                "name": "question",
-                "translations": [
-                  {
-                    "language": "en", "text": ${i18n.en.h_d_development_issues}
```

Comment:
Should we also remove `h_d_development_issues` from the i18n file?

Reply:
I considered but decided it was inconsequential to leave it behind. If you have a strong feeling I should remove I'll be happy to do so, though.